docs/manual/docs/cmd/yammer/network/network-list.md

```diff
@@ -13,11 +13,17 @@ yammer network list [options]
 Option|Description
 ------|-----------
 `--help`|output usage information
-`--includeSuspended true|false`|Include the networks the user is suspended. Default `false`
+`--includeSuspended`|Include the networks the user is suspended.
 `-o, --output [output]`|Output type. `json|text`. Default `text`
 `--verbose`|Runs command with verbose logging
 `--debug`|Runs command with debug logging
 
+## Remarks
```

Comment:
Are these remarks specific to this one command or all Yammer commands? If the latter, then I suggest that we move them to docs where we [discuss authorization and permissions](https://github.com/pnp/office365-cli/blob/master/docs/manual/docs/concepts/authorization-tokens.md) so that we don't need to repeat them in every command

Reply:
This is like this for every Yammer command. Do you want to integrate it there or should I do it? 

What's your preference?

Reply:
@waldekmastykarz: I removed some commands that are not required at all now. The -output parameter will be the same for all Yammer operations.

Reply:
I didn't mean the options. We should always include all options even if some, like debug, verbose and output, are shared across all commands. I meant actually the content of the _Remarks_ sections

Reply:
@waldekmastykarz, I added some text in the file as per your suggestion and removed the remarks from the command so we can merge the command in the current iteration. You can see the text I added [here](https://github.com/pnp/office365-cli/compare/dev...VelinGeorgiev:yammer-network?expand=1) and align when merging with the `dev` branch.